librz/bin/bin.c

```diff
@@ -1466,3 +1239,45 @@ RZ_API RzBinTrycatch *rz_bin_trycatch_new(ut64 source, ut64 from, ut64 to, ut64
 RZ_API void rz_bin_trycatch_free(RzBinTrycatch *tc) {
 	free(tc);
 }
+
+RZ_API const RzBinPlugin *rz_bin_plugin_get(RzBin *bin, const char *name) {
```

Comment:
Add Doxygen, `RZ_NONNULL` for `name`, ownership mark for the return value.

Reply:
I think the `const` specifier is already enough for the return value. What do you think? When you have `const <type>` it is already clear that the caller does not have to free the object (he can't, because the _free function needs a non-const pointer).

Reply:
Ok, fine. But please add `RZ_NONNULL` for `name` then.

Reply:
Header file.